docs/content/preview/explore/change-data-capture/_index.md

```diff
@@ -143,13 +138,27 @@ The snapshot feature uses the `cdc_snapshot_batch_size` GFlag. This flag's defau
 ## Limitations
 
 * YCQL tables aren't currently supported. Issue [11320](https://github.com/yugabyte/yugabyte-db/issues/11320).
+* User Defined Types (UDT) are not supported. Issue [12744](https://github.com/yugabyte/yugabyte-db/issues/12744).
+
+{{< note title="Note" >}}
+
+In the current implementation, information related to the columns for the UDTs will not be there in the messages published on Kafka topic.
+
+{{< /note >}}
+
+* CDC behaviour is undefined on downgrading from a CDC supported version (2.13 and newer) to an unsupported version (2.12 and older) and upgrading it back. Issue [12800](https://github.com/yugabyte/yugabyte-db/issues/12800)
 * DROP and TRUNCATE commands aren't supported. If a user tries to issue these commands on a table while a stream ID is there for the table, the server might crash, the behaviour is unstable. Issues for TRUNCATE [10010](https://github.com/yugabyte/yugabyte-db/issues/10010) and DROP [10069](https://github.com/yugabyte/yugabyte-db/issues/10069).
 * If a stream ID is created, and after that a new table is created, the existing stream ID is not able to stream data from the newly created table. The user needs to create a new stream ID. Issue [10921](https://github.com/yugabyte/yugabyte-db/issues/10921).
+* CDC is not supported on a target table for xCluster replication [11829](https://github.com/yugabyte/yugabyte-db/issues/11829).
+* The support for DDLs is not there yet.
 * A single stream can only be used to stream data from one namespace only.
 * Enabling CDC on tables created using previous versions of YugabyteDB is not supported, even after YugabyteDB is upgraded to version 2.13 or higher.
+* There should be a primary key on the table you want to stream the changes from.
```

Comment:
Isn't this a prerequisite? You moved it from that section, and I assume things won't work right without it...

Reply:
I was asked to move this out of prerequisite to limitations. Do you think I should also mention something similar under prerequisites as well?

Reply:
Yes, I would put it back in prereqs and reword to say something like "Currently, there should be a primary key... This limitation will be removed in a future version of YugabyteDB."